### ISSUE TITLE
i18n checklist for declaring language at the resource level

### ISSUE THREAD

**Issue by takuki** [w3c/wot-thing-description#584]
TD spec defines fields "support", "description", "title" and "name" with which the authors of TD can add natural language texts that can be read by a human.

**Comment by mmccool** [comment]
JSON-LD 1.1 supports use of an `@language` tag in the `@context` object (required in TDs) which sets the default language for the document.  This allows in-band specification of the default language.  In addition, a default language can be requested when a TD is fetched using content negotiation, which supports out-of-band specification of the default language.   The JSON-LD 1.1 spec also allows local `@context` specifications which implies that it would be possible to change the default language even for sections of a TD.

In short, I think we have this well covered by the fact that we support JSON-LD 1.1 and content negotiation.

**Comment by mmccool** [comment]
Note we decided to remove "name" completely (replacing it with title/titles at the Thing level) and require that "support" is an IRI/URI.   So only title/titles and description/descriptions contain human-readable metadata, we support explicit declaration of the language for each of these, and have defined rules for infering the base text direction from the language tags (requiring a script subtag if necessary).

**Comment by mkovatsc** [comment]
We changed `support`to type anyURI.
We removed the special `name` field for Things and use the general `title` mechanism.
Both `title` and `description` have corresponding multi-language fields (ending with plural s) for I18N.
`title` and `description` have metadata through the language tag in `@language`.
`titles` and `descriptions` use objects with a language tag as name.
We provide reviewed rules to handle base directions and provide langauge considerations with pointer to string-meta document.

This allows I18N for all human-readable texts.

Closing
